### PR TITLE
[CI] Use expected accuracy csv files to check benchmark test status

### PR DESCRIPTION
**Pull request by desertfire**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #98840
* __->__ #98839

cc @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx

### CODE REVIEW

File: benchmarks/dynamo/ci_expected_accuracy/update_expected.py [hunk 1]
```diff
@@ -73,10 +73,10 @@ def get_artifacts_urls(results, suites):
 
 
 def normalize_suite_filename(suite_name):
-    assert suite_name.find("inductor_") == 0
     subsuite = suite_name.split("_")[1]
     if "timm" in subsuite:
-        subsuite = f"{subsuite}_models"
+        subsuite = subsuite.replace("timm", "timm_models")
```
Comment (ezyang) [inline benchmarks/dynamo/ci_expected_accuracy/update_expected.py:78]: this is annoying, why can't we have just called the original timm_models

File: benchmarks/dynamo/check_graph_breaks.py [hunk 2]
```diff
@@ -6,17 +6,20 @@
 import pandas as pd
 
 
-def get_field(csv, model_name: str, field: str, typ=float):
-    return typ(csv.loc[csv["name"] == model_name][field])
+def get_field(csv, model_name: str, field: str):
+    try:
+        return csv.loc[csv["name"] == model_name][field].item()
+    except Exception as e:
+        return None
```
Comment (ezyang) [inline benchmarks/dynamo/check_graph_breaks.py:13]: don't suppress all exceptions. Is there a more specific exception type you can catch here?

File: benchmarks/dynamo/check_accuracy.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
+import argparse
+import os
+import sys
+import textwrap
+
+import pandas as pd
```
Comment (ezyang) [inline benchmarks/dynamo/check_accuracy.py:6]: This looks fairly duplicated with check_graph_breaks.py. Can we just fold these two scripts together?

**Comment by ezyang** [comment]
I'm not sure this is an improvement. Skip list is important if the model flakily fails accuracy. Are you sure you've killed all flakiness?

File: benchmarks/dynamo/ci_expected_accuracy/update_expected.py (hunk 1 above)
Comment (desertfire) [reply benchmarks/dynamo/ci_expected_accuracy/update_expected.py:78]: `suite_name` comes from the CI config name, which does not contain the suffix, `_models`. I would vote for doing it the other way, changing `timm_models.py` to  `timm.py`.

**Comment by desertfire** [comment]
> I'm not sure this is an improvement. Skip list is important if the model flakily fails accuracy. Are you sure you've killed all flakiness?

Yes, inductor tests look pretty stable lately. Those flaky tests were likely fixed by triton upgrade.

**Review by ezyang — APPROVED** [review]

**Pull request closed without merging**